hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -1425,7 +1421,7 @@
       name: chip_rv_dm_lc_disabled
       build_mode: "cover_reg_top"
       uvm_test_seq: "chip_rv_dm_lc_disabled_vseq"
-      en_run_modes: ["stub_cpu_mode_with_test_rom"]
+      en_run_modes: ["stub_cpu_mode"]
```

Comment:
ah just to make sure i understand, since ROM is always loaded, we no longer need a `with_test_rom` option, is that right?

Reply:
yes that is correct. I have added this special mode only for that test last week, but it probs makes sense to just use this for all stub CPU sequences so that we are simulating with an environment that resembles the "real" HW more closely.

Reply:
i have no idea if it matters at all in terms of simulation speed. hehe. Probably not important enough to really matter.
I would just merge as is, and if it is better for some reason we can tweak in the future.

Reply:
I provided a suggestion above - the stub CPU mode tests were able to start off immediately. But now, they are also going to be bottlenecked by the bazel step, which enforces serialization across all simulation instances. It is probably not a huge deal (maybe we can benchmark and see). But given the frequent bazel outages, it'd be great to minimize dependencies.